tests/product/test_server_install.py

```diff
@@ -227,24 +227,6 @@ def test_install(self, installer=None):
             self.assert_has_default_config(container)
             self.assert_has_default_connector(container)
 
-    def test_install_worker_is_pa_master(self):
```

Comment:
also, I'm pretty sure that our normal installer doesn't set java8 home so that path is tested in tests anyway